### ISSUE TITLE
[@nrwl/react] Scripts/Sockets run twice

### ISSUE THREAD

**Issue by Ponjimon** [nrwl/nx#3717]
## Current Behavior
I made a very basic React app using `@nrwl/react` and I noticed that the HMR websocket is opened twice.
![image](https://user-images.githubusercontent.com/770632/92996457-00335c00-f50c-11ea-875d-fdd676605bfa.png)

## Expected Behavior
It should only open one socket.

## Steps to Reproduce

1. Clone the [nx-example repo](https://github.com/nrwl/nx-examples).
2. Install dependencies
3. `yarn start cart`
4. Open `http://localhost:4200` with the dev console opened
5. See the above screenshot

### Environment
```
  @nrwl/angular : 10.0.6
  @nrwl/cli : 10.0.6
  @nrwl/cypress : 10.0.6
  @nrwl/eslint-plugin-nx : Not Found
  @nrwl/express : Not Found
  @nrwl/jest : 10.0.6
  @nrwl/linter : 10.0.6
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : Not Found
  @nrwl/react : 10.0.6
  @nrwl/schematics : Not Found
  @nrwl/tao : 10.0.6
  @nrwl/web : 10.0.6
  @nrwl/workspace : 10.0.6
  typescript : 3.9.7
```

But I also tried it with an updated version of `10.2.0` but the behavior was the same.

**Comment by Ponjimon** [comment]
I also added a socket.io socket and it was opened twice too! It also happens if I build the project and serve the compiled files, so it's not just happening when in dev environment.

**Comment by bekos** [comment]
@Ponjimon this should be fixed via 0ec634d61a0a68fe67cb913ed2718bc2f9ec7064, and will be available in the next release.
